File: Samples/JotFormTest/Classes/Controller/GetHistoryViewController.m
//
//  GetHistoryViewController.m
//  JotFormTest
//
//  Created by Curtis Stilwell on 8/21/17.
//  Copyright © 2017 wang. All rights reserved.
//

#import "GetHistoryViewController.h"
#import "SharedData.h"

@interface GetHistoryViewController ()

@property (nonatomic, weak) IBOutlet UITextView *textView;

@end

@implementation GetHistoryViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    [[SharedData sharedData].apiClient getHistory:^(id result) {
     
        self.textView.text = [NSString stringWithFormat:@"%@",result];
  
    } onFailure:^(NSError *error) {
        
    }];
}

@end

File: Samples/JotFormTest/Classes/Controller/GetHistoryViewController.swift
//
//  GetHistoryViewController.swift
//  JotFormTest
//
//  Created by Curtis Stilwell on 2/4/18.
//  Copyright © 2018 wang. All rights reserved.
//

import Foundation

class GetHistoryViewController: UIViewController {
   
    @IBOutlet weak var textView: UITextView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view from its nib.
        SharedData.sharedData.apiClient?.getHistory({(_ result: Any) -> Void in
            self.textView.text = "\(result)"
        }, onFailure: {(_ error: Error?) -> Void in
        })
    }
}
